automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -214,7 +214,7 @@ def product_upgrade(product):
                                 upgraded = str(execute(
                                     get_sat_cap_version, 'cap', host=cap_host
                                     )[cap_host])
-                                if current:
+                                if current != 'None':
```

Comment:
actually the current value will be passed to LooseVersion function to compare previuos version and upgraded version. So while fetching the version I get it string type(as https://github.com/jyejare/automation-tools/blob/dcfd2f1f84a6d3472ff77be58fade467b82f20a7/automation_tools/satellite6/upgrade/__init__.py#L182). 
So that I need not to type cast it everytime(e.g https://github.com/jyejare/automation-tools/blob/dcfd2f1f84a6d3472ff77be58fade467b82f20a7/automation_tools/satellite6/upgrade/__init__.py#L190)